be/src/olap/tablet.cpp

```diff
@@ -205,6 +207,14 @@ WriteCooldownMetaExecutors::WriteCooldownMetaExecutors(size_t executor_nums)
     }
 }
 
+WriteCooldownMetaExecutors::~WriteCooldownMetaExecutors() {
```

Comment:
1. Refactor this code, add WriteCooldownMetaExecutors to exec_env.
2. Add a shutdown method to WriteCooldownMetaExecutors and call shutdown in exec_env's shutdown.